src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/NuGetServiceMessagePackRpcDescriptor.cs

```diff
@@ -34,11 +34,14 @@ static NuGetServiceMessagePackRpcDescriptor()
                 PackageSourceFormatter.Instance,
                 PackageSourceUpdateOptionsFormatter.Instance,
                 ProjectActionFormatter.Instance,
+                RemoteErrorFormatter.Instance,
```

Comment:
ILogMessageFormatter.Instance?

Reply:
It's not generally needed; we don't pass raw `ILogMessage` types anywhere outside of the `RemoteError` type, and the `RemoteErrorFormatter` uses `ILogMessageFormatter.Instance` internally.